public/app/plugins/datasource/mixed/datasource.ts

```diff
@@ -17,17 +17,13 @@ class MixedDatasource extends DataSourceApi<DataQuery> {
         return Promise.resolve([]);
       }
 
-      const filtered = _.filter(targets, (t: DataQuery) => {
-        return !t.hide;
-      });
-
-      if (filtered.length === 0) {
+      if (targets.length === 0) {
         return { data: [] };
       }
 
       return this.datasourceSrv.get(dsName).then(ds => {
         const opt = _.cloneDeep(options);
-        opt.targets = filtered;
```

Comment:
Should this not filter if hiddenQueries is false?

Reply:
yup -- updated